plutus-book/doc/03-UTxO.adoc

```diff
@@ -75,7 +76,7 @@ For the basic transaction
 model we discuss here, refer to <<utxo-definitions>>.
 
 [#utxo-definitions]
-.Basic UTxO definitions
+// .Basic UTxO definitions
```

Comment:
If you do this then the link to it will have no reftext. It was redundant before, but it looks like you've cropped the caption out of the figure which is much better. So I don't think this should be commented out.

Reply:
it is the only figure with a caption - should i leave the adoc caption in anyway?